tests/sandbox/conftest.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+import json
+import requests
+
+from typing import Dict, Callable
+
+from data import Actor, RenamedHeader
+from utils import HttpMethod
+
+
+@pytest.fixture(scope="session")
+def sandbox_url(environment, service_url) -> str:
+    """Provides an overload for the standard service_url fixture to support running the sandbox against a local deployment"""
+    return "http://127.0.0.1:9000" if environment == "local" else service_url
+
+
+@pytest.fixture(scope="session")
+def load_json(environment) -> Callable[[str], str]:
+    """Provides a method which encapsulates the logic of retrieving some example JSON from a file"""
+    base_path = (
+        "sandbox"
+        if environment == "local"
+        else "proxies/sandbox/apiproxy/resources/hosted"
+    )
+    return lambda path: _get_json(base_path + "/src/mocks/", path)
+
+
+@pytest.fixture
+def send_rest_request(
+    sandbox_url,
+) -> Callable[[HttpMethod, str, Actor], requests.Response]:
+    """Provides a method which encapsulates the logic of sending a REST call to the correct base URL"""
+    return lambda method, url, actor, headers={}, **kwargs: _send_rest_request(
```

Comment:
Same comment about lambda. I think this is redundant. Private methods in Python are not really Java-private, so I think this is unnecessary,